src/scripts/shell/dimPlatformChoice.component.js

```diff
@@ -0,0 +1,35 @@
+import templateUrl from './dimPlatformChoice.html';
+import './dimPlatformChoice.scss';
+
+const PlatformChoice = {
+  bindings: {
+    currentPlatform: '<',
+    onPlatformChange: '&',
+    platforms: '<'
+  },
+  controller: function PlatformChoiceCtrl() {
+    var self = this;
+
+    self.currentPlatform = null;
+    self.platforms = [];
+
+    self.$onChanges = (changes) => { // eslint-disable-line no-unused-vars
```

Comment:
At one point, they were functions that were standalone.  I added the disable flag.  Then I made they apart of the instance.  I'll remove them.

![2017-01-25](https://cloud.githubusercontent.com/assets/984608/22281957/42ee3df4-e29f-11e6-88cb-4138985dd1ce.png)